src/store/modules/GlobalStore.js

```diff
@@ -150,6 +150,18 @@ const GlobalStore = {
         })
         .catch((error) => {
           console.log(error);
+          return dispatch('getAccountService');
+        });
+    },
+    getAccountService() {
+      return api
+        .get('/redfish/v1/AccountService')
+        .then((response) => {
+          if (response.data?.LDAP?.RemoteRoleMapping?.length > 0) {
+            return Promise.resolve();
+          }
```

Comment:
Can we add an else condition where we will reject with proper message? Promise.reject('No remote role mappings found'); if condition fails we will get undefined, which is not an error or a rejected promise.

Reply:
No, We will not reject, if the API is passing. It is in then block, so, it will be resolve by default.
We will only reject if the API is failing.

Reply:
Agreed!